.github/workflows/release.yml

```diff
@@ -121,6 +125,7 @@ jobs:
       - integration_test
       - unit_test
     runs-on: ubuntu-latest
+    if: jobs.prepare-release.outputs.release_status == 'unreleased'
```

Comment:
While in the same workflow (which had a previously unreleased Sentry version) we can re-run this step without it short-circuiting here, right?
While I like the simplicity of re-using the Sentry state of the release – I'm wondering whether there are edge cases where we could lock ourselves out of retrying a release – but I suppose that might happen with any way of storing this "released?" state 🤔

Reply:
I think that's a reasonable issue with the current way. Unfortunately, I don't know how to best solve it :/. I think what we have right now is the ability to release and let npm / pypi etc. deal with the duplication. 

The alternative would be to go into sentry and delete the release before retrying? I don't know if that's possible or how to do it though.

Reply:
@ansgarm since you had a recent experience with this, do you think we can document the sentry release deletion process to unblock this?

Reply:
Yeah!
It's basically this:
```
# Install Sentry CLI and login
npm i -g @sentry/cli
sentry-cli login
# List all releases (optional)
sentry-cli releases list --org hashicorp
# Delete the release, Note: there will be no confirmation for deleting the release!
sentry-cli releases delete --org hashicorp <release> # e.g. cdktf-cli-0.14.0
```

Reply:
Actually, I was doing some further research on the sentry-cli commands that we use, and I realize that we don't need this. I've added it anyway since it's good context for situations like we had earlier, but it's not relevant to the current PR. 

<img width="736" alt="image" src="https://user-images.githubusercontent.com/573531/203554860-b3a2d420-5deb-4bdc-98a3-00fa97480015.png">